packages/api-plugin-promotions/src/simpleSchemas.js

```diff
@@ -81,7 +81,8 @@ export const Promotion = new SimpleSchema({
     type: String
   },
   "description": {
-    type: String
+    type: String,
+    optional: true
```

Comment:
If it's going to be optional here, shouldn't it also be optional in the [GraphQL schema](https://github.com/reactioncommerce/reaction/blob/e73f2c972894201f3a3d77626a43f4a873381422/packages/api-plugin-promotions/src/schemas/schema.graphql#L113)?